tests/sentry/event_manager/test_event_manager.py

```diff
@@ -2454,29 +2452,17 @@ def test_perf_issue_slow_db_issue_not_created(self):
                 event = manager.save(self.project.id)
                 last_event = event
 
-            # The group should not be created, but there should be a tag on the transaction
-            assert len(last_event.groups) == 0
-            transaction.set_tag.assert_called_with("_will_create_slow_db_issue", "true")
-
-    @override_options(
-        {
-            "performance.issues.slow_span.problem-creation": 1.0,
-            "performance_issue_creation_rate": 1.0,
-        }
-    )
-    @override_settings(SENTRY_PERFORMANCE_ISSUES_REDUCE_NOISE=False)
-    def test_perf_issue_slow_db_issue_not_created_with_noise_flag_false(self):
-        self.project.update_option("sentry:performance_issue_creation_rate", 1.0)
+            return last_event
 
-        last_event = None
+        # Should not create the group without the feature flag
+        last_event = attempt_to_generate_slow_db_issue()
+        assert len(last_event.groups) == 0
 
-        for _ in range(100):
-            manager = EventManager(make_event(**get_event("slow-db-spans")))
-            manager.normalize()
-            event = manager.save(self.project.id)
-            last_event = event
+        with self.feature({"organizations:performance-slow-db-issue": True}):
+            last_event = attempt_to_generate_slow_db_issue()
 
-        assert len(last_event.groups) == 0
+            assert len(last_event.groups) == 1
+            assert last_event.groups[0].type == GroupType.PERFORMANCE_SLOW_SPAN.value
```

Comment:
IMO this test is redundant, but up to you. It's testing that the feature flag is respected, which is in the unit test above. It's also testing that the 100 threshold is respected, which is tested thoroughly in the same file, a few specs up

Reply:
You're right, these code paths are already being tested but I wrote this for extra assurance :p

I suppose there's no need for this anymore though.